frontend/src/app.tsx

```diff
@@ -242,6 +243,15 @@ const router = createBrowserRouter([
                             path: ['open-content-management']
                         }
                     },
+                    // {
```

Comment:
was this supposed to be commented out?

I see in staging this is now a live route